include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -74,16 +73,22 @@ def FModuleOp : FIRRTLOp<"module", [IsolatedFromAbove, FunctionLike, Symbol,
   ];
 
   let extraClassDeclaration = [{
-    using FunctionLike::end;
-    using FunctionLike::front;
-    using FunctionLike::getBody;
+    Region &getBody() { return getRegion(); }
+    Block *getBodyBlock() { return &body().front(); }
```

Comment:
I'd recommend dropping the region thing and just have `getBody()` return the block.  FunctionLike was in the way of this before

Reply:
I dropped the `back()` call, no one was using it and it doesn't make sense 🤦 .  As for renaming `getBodyBlock()` -> `getBody()` I'd like to do it in a follow-up commit

Reply:
PR here: https://github.com/llvm/circt/pull/1940